code/modules/projectiles/gun_attachables.dm

```diff
@@ -2386,6 +2386,37 @@ Defined in conflicts.dm of the #defines folder.
 /obj/item/attachable/stock/type71/New()
 	..()
 
+/obj/item/attachable/stock/m60
+	name = "M60 stock"
+	desc = "This isn't supposed to be separated from the gun, how'd this happen?"
+	icon = 'icons/obj/items/weapons/guns/attachments/stock.dmi'
+	icon_state = "m60_stock"
+	attach_icon = "m60_stock"
+	slot = "stock"
+	wield_delay_mod = WIELD_DELAY_NONE
+	flags_attach_features = NO_FLAGS
+	melee_mod = 15
+	size_mod = 0
+
+/obj/item/attachable/stock/m60/New()
+	..()
```

Comment:
despite this file being full of them, these new overrides are pointless, delete it